libs/vkd3d/command.c

```diff
@@ -4790,7 +4794,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_SOSetTargets(d3d12_command_list
 
     TRACE("iface %p, start_slot %u, view_count %u, views %p.\n", iface, start_slot, view_count, views);
 
-    d3d12_command_list_end_current_render_pass(list);
+    d3d12_command_list_end_current_render_pass(list, true);
```

Comment:
Does this need to end the render pass?

Reply:
I think this is being done so that we don't change transform feedback buffers while transform feedback is active.

If we were to remove this, we'd have to revisit Begin/EndTransformFeedback as well.

Reply:
hm, ok.